sdk/tests/invariant.test.ts

```diff
@@ -55,4 +56,83 @@ describe('invariant', function () {
 
     assert.deepEqual(newFee, { v: 100 })
   })
+
+  it('should add fee tier', async () => {
+    const { api, account } = await init()
+
+    const invariantData = await getDeploymentData('invariant')
+    const invariant = new Invariant(api, Network.Local)
+
+    const initFee = { v: 10 }
```

Comment:
`10` is not a realistic value. It's better to perform tests on a more representative value. Use `1%`